gadgets/trace_capabilities/gadget.yaml

```diff
@@ -0,0 +1,83 @@
+name: trace capabilities
+description: trace security capabilitiy checks
+homepageURL: https://inspektor-gadget.io/
+documentationURL: https://inspektor-gadget.io/docs
+sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+tracers:
+  capabilities:
+    mapName: events
+    structName: cap_event
+structs:
+  cap_event:
+    fields:
+    - name: mntnsid
+      description: mount namespace inode id
+      attributes:
+        template: ns
+    - name: current_userns
+      attributes:
+        template: ns
+    - name: target_userns
+      attributes:
+        template: ns
+    - name: cap_effective
+      attributes:
+        width: 20
+        alignment: left
+        ellipsis: end
```

Comment:
Once https://github.com/inspektor-gadget/inspektor-gadget/pull/3020 is merged, we can print cap_effective in a better way.

(not for this PR)